docs/eigenlayer/avs-guides/spec/_category_.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
This section should be "AVS Guides / Node Spec" as opposed to "AVS Guides / Spec"

Reply:
no problem, could you clarify what you mean by "Node Spec" and Spec"?

Currently the label of this page is "AVS Node Specification" under "AVS Guides" page. However, we can change the title of the page "Node Specification" if you prefer?

Reply:
Oh I was just referring to the file path `docs/eigenlayer/avs-guides/spec/_category.json`

If it shows up as "Node Specification" under "AVS Guides" which it does I think we're good.